tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/MacroSuite.scala

```diff
@@ -0,0 +1,169 @@
+package scala.meta.tests.parsers.dotty
+
+import scala.meta.tests.parsers._
+import scala.meta._
+
+class MacroSuite extends BaseDottySuite {
+
+  implicit val parseBlock: String => Stat = code => blockStat(code)(dialects.Dotty)
+
+  /** All examples based on dotty documentation:
+   *  https://dotty.epfl.ch/docs/reference/metaprogramming/toc.html
+   *  https://dotty.epfl.ch/docs/reference/metaprogramming/macros.html
+   *  https://dotty.epfl.ch/docs/reference/metaprogramming/tasty-reflect.html
+   *
+   *  '{ ... } OR 'ident - QuotedMacroExpr
+   *  '[ ... ] - QuotedMacroType
+   *  ${ ... } OR $ident - SplicedMacroExpr
+   *  symbols 'ident are not supported in dotty
+   */
+  test("parse-single-quote-character") {
+    runTestAssert[Stat]("val a = 'c'")(
+      Defn.Val(Nil, List(Pat.Var(tname("a"))), None, Lit.Char('c'))
+    )
+  }
+
+  test("macro-quote-expr") {
+    runTestAssert[Stat]("'{ Int.MinValue }")(
+      Term.QuotedMacroExpr(List(Term.Select(tname("Int"), tname("MinValue"))))
+    )
+    runTestAssert[Stat]("'{ (x: T) => 'x }")(
+      Term.QuotedMacroExpr(List(Term.Function(List(tparam("x", "T")), Lit.Symbol('x))))
```

Comment:
Should `'x`  not be a `QuotedMacroExpr`  and not `Lit.Symbol`? Or should it be a least the same x?

Reply:
Yes it should be `QuotedMacroExpr` good catch :+1:

Reply:
fixed